dockerfiles/init/modules/keycloak/templates/che-realm.json.erb

```diff
@@ -60,8 +60,8 @@
     "enabled" : true,
     "clientAuthenticatorType" : "client-secret",
     "secret" : "2c1b2621-d251-4701-82c4-a7dd447faa97",
-    "redirectUris" : [ "*" ],
-    "webOrigins" : [ "*" ],
+    "redirectUris" : [ "<%= scope.lookupvar('che::che_server_url') %>/*" ],
```

Comment:
Do we have one client here or two? ```che-public```  or ```che``` and ```che-public```?

Reply:
I haven't rebased on your last commits where you removed one of the clients.
I'll do it just after and one the clients will probably be removed